bin/cv_regress

```diff
@@ -151,6 +151,8 @@ def read_file(args, file):
             test.sub_make(args.make)
         if args.iss != None:
             test.iss = args.iss
+        else:
+            args.iss = 'YES'
```

Comment:
Does this make USE_ISS=YES the default if no `--iss` is given?

Reply:
for the build-phase yes, not the run-phase (which uses the per-test setting)